README.md

```diff
@@ -6,52 +6,66 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
+**4 - Showing the Compiled Python:** Checking the box labeled `PYTHON` will show the user the compiled Python, which is the output of the Coconut compiler. the Python code will be displayed using Python syntax highlighting.
 
-On the left, we can use the code editor. The editor allows for the keyboard shortcuts from Ace editor (listed [here](https://github.com/ajaxorg/ace/wiki/Default-Keyboard-Shortcuts)). In addition, the editor includes Python and Coconut syntax highlighting for all keywords (Coconut specific keywords listed [here](http://coconut.readthedocs.io/en/latest/DOCS.html#keywords)):
+**5 - Code Editor:** The code editor, which is on the left side of the webpage, is where the user types in their Coconut code. The editor has line numbers and Coconut-specific syntax highlighting.
 
-![coco specific](https://user-images.githubusercontent.com/35832643/37873781-17561450-2fd8-11e8-8659-1c70463a48aa.png)
+**6 - Code Execution Output:** The code execution output is on the right side of the webpage. 
 
-Clicking run will compile and execute the program. The output will show up on the right. If there is a compiling error, the output will display the error from the Coconut compiler along with the traceback highlighted in red in the user's original code:
+Furthermore, if there is an error in compilation or execution, the output will display the traceback. If the error came from the Coconut compiler, the offending Coconut line will be highlighted in red in the code editor:
 
-![red tracebacks](https://user-images.githubusercontent.com/35832643/37873784-28079e7c-2fd8-11e8-8c7e-14a954f44264.png)
+![error2](https://user-images.githubusercontent.com/35832643/38785410-89d90052-40d4-11e8-93c5-8cec8fea532c.png)
 
-After the code runs, the user's code is stored in the session, so they can make changes to their existing code. 
+If the error occurred at runtime, then the offending line will be highlighted in red in the Python code:
 
-The interpreter also supports multiple lines of output:
+![error1](https://user-images.githubusercontent.com/35832643/38785409-89c18620-40d4-11e8-8a52-19150efae3ea.png)
 
-![example code](https://user-images.githubusercontent.com/35832643/37873786-338affdc-2fd8-11e8-8544-cb8c85bd9be3.png)
 
-The user has the option of seeing the compiled Python code by clicking the box next to Python at the top right corner:
 
-![python code](https://user-images.githubusercontent.com/35832643/37873787-43191844-2fd8-11e8-90b0-e58317c27420.png)
 
-The user is also able to see tracebacks highlighted in red in the Python code as well:
-![python tracebacks](https://user-images.githubusercontent.com/35832643/37873790-49e9130e-2fd8-11e8-9bb1-2f3467d09393.png)
+## Known Problems
+Currently, there are no known problems. However, on the current deployment, users cannot choose the Coconut version on the interpreter except the version installed by developers. The developer team plans to address this issue in the future.
 
-# Issues
+## Contributing
```

Comment:
Refer to CONTRIBUTING.md for contributing. The file may need to be modified a little, though.